src/utils/enums.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable */
```

Comment:
IDK but it was giving me lint errors to use default while exporting, but all Good. :)